src/main/scala/org/labrad/manager/Listener.scala

```diff
@@ -12,42 +14,65 @@ import scala.concurrent.ExecutionContext
 /**
```

Comment:
Hmm, I'd say keep it to one line - closer to java and no need to think "is this the situation that I have to use two blanks lines or one"